core/src/main/java/ai/timefold/solver/core/api/score/analysis/ScoreAnalysis.java

```diff
@@ -53,6 +55,8 @@
 public record ScoreAnalysis<Score_ extends Score<Score_>>(Score_ score,
```

Comment:
Also, do we want to make it simple to figure out that the score was not initialized?
To find out today, you have to parse the score and check for `init`.
Maybe we want to provide a new field which does this on behalf of the user?